### PR TITLE
added the load failed message

### LINKED ISSUE #9684: If BokehJS isn't available, error message should be displayed
I'm trying to use Bokeh behind a proxy that doesn't allow outbound connections to https://cdn.pydata.org, which means that BokehJS can't be downloaded.  I can't do bokeh.resources = inline, as I have been unable to install bokehjs and its dependencies using npm due to the same lack of connectivity.

When I try to display a figure, nothing happens, but when I inspect the element, I can see the root cause is that BokehJS is not available (screenshot below).  I am getting pydata.org whitelisted, as we are not able to whitelist github and therefore it is challenging to get npm dependencies to install properly for local installation.  I think this should resolve the issue, but is it possible to have an error message at this point instead of just not rendering the script.

Alternative would be to have jupyter-bokeh include the installation of bokehjs locally, but I don't know the rationale behind having it loaded from the CDN, so assume it doesn't do that for a specific reason.

OS: Amazon Linux 
Browser: Firefox 71.0
Python: 3.6.8
Jupyter lab version: 1.2.4
Bokeh: 1.3.4
jupyter-bokeh: 1.1.1

Screen Shot 2019-12-20 at 12 45 29 pm

### PR DESCRIPTION
**Pull request by ayushiitr2901**
This Pull Request is regarding issue #9684, "Failed to load" message is visible now if BokehJS does not load for any possible reason. I had a copy and paste the exact code in a new repository that is why it seems the whole code is altered, whereas the only changes made are in line 166,167 and 168.
Please ignore the rest, it is the same code as previously.
- [ ] issues: fixes #9684 
- [ ] tests added / passed
I have tested in my local virtual env.

### CODE REVIEW

File: bokeh/core/_templates/autoload_nb_js.js [hunk 1]
```diff
@@ -1,186 +1,189 @@
...
-      setTimeout(run_inline_js, 100);
-    } else if (!root._bokeh_failed_load) {
-      console.log("Bokeh: BokehJS failed to load within specified timeout.");
-      root._bokeh_failed_load = true;
-    } else if (force !== true) {
-      var cell = $(document.getElementById({{ elementid|json }})).parents('.cell').data().cell;
-      cell.output_area.append_execute_result(NB_LOAD_WARNING)
-    }
-{% endblock %}
+	{% extends "autoload_js.js" %}
```
Comment (mattpap) [inline bokeh/core/_templates/autoload_nb_js.js]: Please restore original indentation and any other formatting changes.

**Comment by bryevdv** [comment]
@ayushiitr2901 A few tips:

* You have made a PR from the `master` branch of your repository. It's generally advised to make separate branches on your fork of the repo, and then make the PRs from those branches, not from `master`. The reason is that it is harder to undo changes on your fork if you need to start over, or if a PR is declined. 

* As @mattpap mentioned, the PR diff should only contain relevant changes. The current unnecessary formatting and whitespace changes make the diff harder to read, but also are failing our codebase style/lint checks

* You will also need to update `tests/unit/bokeh/core/test_templates.py`. There is a test there for the hash of the template. It's not really to test the functionality but was put there as a extra layer of protection against the template being change without intention and review.

**Comment by ayushiitr2901** [comment]
Thanks for the tips. I made some very common mistakes and I apologize for the same. I will now work on it make a PR again.

**Comment by bryevdv** [comment]
superseded by #9718

**Pull request closed without merging**